frontend/src/metabase/entities/questions.js

```diff
@@ -26,6 +27,62 @@ const Questions = createEntity({
   nameOne: "question",
   path: "/api/card",
 
+  /**
+   * Temporarily mock endpoints for Metrics v2
+   *
+   * Any question with `type: "metric"` will be passed to API as `type: "question"`.
+   * Same goes for any questions with a name starting with "Metric" (case-insensitive).
+   */
+  api: {
+    get: async payload => {
+      const get = GET("/api/card/:id");
+      const result = await get(payload);
+
+      if (result.name.toLowerCase().startsWith("metric")) {
+        return { ...result, type: "metric", dataset: false };
```

Comment:
`dataset: false` is still only a temporary measure during the migration period, right?

Reply:
Hopefully. We will have both `dataset` and `type`, but we should not be using `dataset` anymore.